example/lib/main.dart

```diff
@@ -37,38 +40,36 @@ class _MyAppState extends State<MyApp> implements IAcessoBio, IAcessoBioCamera,
 
   Future<void> initAcessoBio() async {
     acessobio = new UnicoCheck(
-        this,
-        acessoPass.url,
-        acessoPass.apikey,
-        acessoPass.token
+      this,
+      'acessoPass.url',
+      'acessoPass.apikey',
+      'acessoPass.token',
```

Comment:
Já que essa variável `acessoPass` não existe, seria legal remover essa parte da String para ficar um pouco mais claro do que precisa passar para o UnicoCheck()
```suggestion
      'url',
      'apikey',
      'token',
```

Reply:
Na PR https://github.com/acesso-io/unico_check_flutter/pull/4 já está dessa forma.

É a continuação das modificações dessa daqui.